### DISCUSSION TITLE
建议添加 error code 标准

### DISCUSSION THREAD

**Discussion by shigma** [howmanybots/onebot#78] [想法 / 建议]
目前各平台实现对 API 的 error code 没有统一的标准（指具体对应到错误原因），建议在未来的版本中统一；或者在调用失败的返回值中加入 message 字段，以区分不同的错误类型。

举个例子，send_group_msg 可能的错误原因包括：

- 无法发包
- 被禁言
- 图片加载失败
- ……

**Comment by sdy623** [comment]
还有，CQ码转换异常等等

**Comment by sdy623** [comment]
建议错误码也整理成API，实现插件自己的异常处理。
